modyn/supervisor/internal/grpc_handler.py

```diff
@@ -66,20 +66,30 @@
 class GRPCHandler:
     # pylint: disable=too-many-instance-attributes
 
+    # pylint: disable=unused-argument
     def __init__(
         self,
         modyn_config: dict,
         progress_mgr: Optional[enlighten.Manager] = None,
         status_bar: Optional[enlighten.StatusBar] = None,
```

Comment:
Do you plan to remove this within a follow PR? I am fine with either (removing it now or in a follow up PR)

Reply:
removed.

Reply:
Thanks